dockers/docker-orchagent/orchagent.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Export platform information. Required to be able to write
+# vendor specific code.
+export platform=`sonic-cfggen -m /etc/sonic/minigraph.xml -v onie_switch_asic`
```

Comment:
Hi @oleksandrivantsiv , Could you create a pull request to switch the code in swss to use mellanox instead? I think that could be a cleaner solution. After that we could clean up code here.